### PR TITLE
Prep for codeclimate

### PR DESCRIPTION
**Pull request by cromwellryan**
Codeclimate has suggested we might be able to get our config added to their ESLint engine. While I'm a little dubious this will get accepted, I was gonna try to get this thing all shiny to avoid any weird looks when they check the PR 😄  ✨

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+`npm install --save-dev eslint-config-sparkbox` or `yarn add eslint-config-sparkbox`
+
+Use [extends syntax] in your `.eslintrc` file:
+
+    {
+      "extends": ["sparkbox"]
+    }
+
+_Note:_ Depending on your editor setup you may need the following dependencies installed globally (~-g~) through `npm`.
```
Comment (iAmNathanJ) [inline README.md]: You want those tildes there around the -g?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+* [React](https://github.com/yannickcr/eslint-plugin-react)
+* [Flowtype](https://github.com/gajus/eslint-plugin-flowtype)
+* [JSX a11y](https://github.com/evcohen/eslint-plugin-jsx-a11y)
+* [ES6+ imports](https://github.com/benmosher/eslint-plugin-import)
+
+## Usage
+
+`eslint-config-sparkbox` is designed as an [ESLint Sharable Config]
+
+`npm install --save-dev eslint-config-sparkbox` or `yarn add eslint-config-sparkbox`
```
Comment (iAmNathanJ) [inline README.md]: Since we have a yarn.lock, should we recommend `yarn add` first and `npm i` secondly?

**Comment by asimpson** [comment]
Love this!

Switching to `devDependencies` may cause issues with continuous integration systems no?

**Comment by cromwellryan** [comment]
@asimpson Good catch. I'm not sure. CI runs in empty NODE_ENV mode, no? What's the easiest way to test this? Create a project probably?

File: README.md (hunk 2 above)
Comment (cromwellryan) [inline README.md]: Yeah - probably.

**Pull request merged**